requirements/requirements.in

```diff
@@ -1,5 +1,5 @@
 certifi
-elasticsearch
+elasticsearch==6.3.1
```

Comment:
Upgrading this is going to have be part of the work of upgrading ES
or https://github.com/hypothesis/bouncer/issues/645